BlueShift-iOS-SDK/BlueShift.m

```diff
@@ -208,22 +210,24 @@ - (void) setupWithConfiguration:(BlueShiftConfig *)config {
             }
         }
         
-        [self setupObservers];
-        
-        [self logSDKInitializationDetails];
-        
-        // Fire app open if device token is already present, else delay it till app receives device token.
-        if ([self getDeviceToken]) {
-            [_sharedBlueShiftInstance.appDelegate trackAppOpenOnAppLaunch:nil];
-        }
-        
-        // Send any existing cached non batch/track events to Blueshift irrespecitive of SDK Tracking enabled status
-        [BlueShiftRequestQueue processRequestsInQueue];
+        [self runAfterSDKInitialisation];
     } @catch (NSException *exception) {
         [BlueshiftLog logException:exception withDescription:@"Failed to initialise SDK." methodName:[NSString stringWithUTF8String:__PRETTY_FUNCTION__]];
     }
 }
 
+- (void)runAfterSDKInitialisation {
+    [self logSDKInitializationDetails];
+
+    [self setupObservers];
```

Comment:
You changed the order of the methods above. Does that have any significance?

Reply:
The is no significance of order here, its just a log will be printed before setting observers.